shell/common/shell_unittests.cc

```diff
@@ -952,5 +953,69 @@ TEST_F(ShellTest, IsolateCanAccessPersistentIsolateData) {
   DestroyShell(std::move(shell), std::move(task_runners));
 }
 
+TEST_F(ShellTest, Screenshot) {
```

Comment:
I guess this test would not fail before this PR as it's not setting up a Fuchsia embedder? Maybe add a comment here to clarify that, and say that we'll eventually want to have a Fuchsia unit test.

Reply:
It's more that we don't run shell_unittests yet on Fuchsia. It's on the roadmap, so it's more that the entire test suite needs to be enabled rather than this test or having a specific Fuchsia unit test.